cas-hashable/cas-hashable.cabal

```diff
@@ -0,0 +1,47 @@
+Name:                cas-hashable
+Version:             1.0.0
+Synopsis:            A hashing class for content-addressed storage
+Description:
+            A hashing class for content-addressed storage
+License:             MIT
+License-file:        LICENSE
+Author:              Tom Nielsen, Nicholas Clarke, Andreas Herrmann
+Maintainer:          nicholas.clarke@tweag.io
+build-type:          Simple
+Cabal-Version:       >= 1.10
+homepage:            https://github.com/tweag/funflow
+bug-reports:         https://github.com/tweag/funflow
+category:            Control
+Tested-With:         GHC == 7.8.4, GHC == 7.10.2, GHC == 7.10.3, GHC == 8.0.1
+
+extra-source-files:
+                   changelog.md
+
+Library
+   ghc-options:       -Wall -fno-warn-type-defaults
```

Comment:
Maybe that's already the case for the other funflow libs, but I don't think that putting `-Wall` in the cabal file is a very good idea (because downstream users don't want warnings in one of their dependency to break their whole build)

Reply:
`-Wall` is not a problem. I think that you were thinking of `-Werror`, which, quite appropriately, doesn't appear to be here.

Reply:
Oh indeed, thanks